cumulus_library/__init__.py

```diff
@@ -17,4 +17,4 @@
 # is all code level. See template_sql for more information.
 
 __all__ = ["BaseTableBuilder", "CountsBuilder", "StudyConfig", "StudyManifest", "get_template"]
-__version__ = "4.5.0"
+__version__ = "4.5.1"
```

Comment:
We're only removing this from the `discovery` study - I know that modifying `core` would be a much more invasive problem, but just noting that these codes would exist there too.

Reply:
core handles this by allowlisting codes in places we care about them - i'm less worried about that

Reply:
I know we allow list in a couple places, but there's lots of places and resources we just denormalize codeable concepts and toss in there. I think Observation codes for one.

Reply:
(I'm not saying we necessarily have to fix this issue there - just noting it. The official recommendation can just be "rerun your ETL")